CSS/layout.css

```diff
@@ -0,0 +1,81 @@
+/*******  CSS/LAYOUT ********/
+
+body {
+  box-sizing: border-box;
+  width: 85%;
+  margin: 0 auto;
+  padding-top: 3em;
+}
+
+
+/*******  HEADER/ NAV  ********/
+
+.flex-header {
+    display:flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
```

Comment:
I don't see there being a problem with it having 'wrap' as long as you gave some bottom margins to the first two flex items.